ui/components/MeshAdapterConfigComponent.js

```diff
@@ -485,6 +485,7 @@ const MeshAdapterConfigComponent = (props) => {
               <Button
                 type="submit"
                 variant="contained"
+                data-test="undeploy-button"
```

Comment:
same here switch to data-testId. Also curious if we can add component name in the id some how something like `data-testid="adapter-undeploy-button"`